members-service/updateMember.js

```diff
@@ -17,21 +17,28 @@ const logger = new OperationsLogger({
 });
 
 export const handlerFunc = (event: any, context: any, callback: any) => {
+  let payload;
```

Comment:
can't you use `const` and a ternary operator?

Reply:
yeah, that's definitely a better way